spec/workers/every_morning_worker_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'fugit'
+
+RSpec.describe Schedule::EveryMorningWorker, type: :worker do
+  let(:time) { (Time.zone.tomorrow + 8.hours).to_datetime }
+  let(:scheduled_job) { described_class.perform_at(time) }
+
+  describe 'testing worker' do
+    it 'is enqueued in the schedul queue' do
+      described_class.perform_async
+
+      assert_equal :schedule, described_class.queue
```

Comment:
we do not currently use "assert" here but "expect"

you might use the matcher `be_processed_in` like an example below
```
it { is_expected.to be_processed_in :default }
```